packages/api/src/external/carequality/document/__tests__/create-outbound-document-retrieval-req.test.ts

```diff
@@ -164,3 +175,95 @@ describe("outboundDocumentRetrievalRequest", () => {
     expect(res[0].samlAttributes.homeCommunityId).toEqual(organization.oid);
   });
 });
+
+describe("correct responses with multiple outboundDocumentQueryResps, where the doc refs have the same homeCommunityId as their respective gateways", () => {
+  it("returns correct data for each query response", async () => {
+    const homeCommunityId1 = faker.string.uuid();
+    const homeCommunityId2 = faker.string.uuid();
+    const outboundDocumentQueryResps: OutboundDocumentQueryResp[] = [
+      makeOutboundDocumentQueryResp({
+        gateway: makeXcaGateway({ homeCommunityId: homeCommunityId1 }),
+        documentReference: [
+          makeDocumentReferenceWithMetriportId({ homeCommunityId: homeCommunityId1 }),
+          makeDocumentReferenceWithMetriportId({ homeCommunityId: homeCommunityId1 }),
+        ],
+      }),
+      makeOutboundDocumentQueryResp({
+        gateway: makeXcaGateway({ homeCommunityId: homeCommunityId2 }),
+        documentReference: [
+          makeDocumentReferenceWithMetriportId({ homeCommunityId: homeCommunityId2 }),
+          makeDocumentReferenceWithMetriportId({ homeCommunityId: homeCommunityId2 }),
+        ],
+      }),
+    ];
+
+    const res: OutboundDocumentRetrievalReq[] = createOutboundDocumentRetrievalReqs({
+      patient,
+      requestId,
+      initiator,
+      outboundDocumentQueryResps,
+    });
+
+    expect(res).toBeTruthy();
+    expect(res.length).toEqual(2);
+
+    expect(res[0]?.documentReference?.length).toEqual(2);
+    expect(res[0]?.documentReference).toEqual(outboundDocumentQueryResps[0]?.documentReference);
+
+    expect(res[1]?.documentReference?.length).toEqual(2);
+    expect(res[1]?.documentReference).toEqual(outboundDocumentQueryResps[1]?.documentReference);
+  });
+  it("correct responses with multiple outboundDocumentQueryResps, where the doc refs for one gateway have different home community ids", async () => {
```

Comment:
The body of this test is exactly the same as the above.

Reply:
fixed the test to use homeCommunityId3. Good catch!